### ISSUE TITLE
Acceptance Test AT2-1

### ISSUE THREAD

**Issue by johnstonstott** [MaelSemler/mini-cap#104]
**Acceptance test:** 

AT2-1

**Feature:** 

FT2-1

**Feature description:** 

As a user I want to pick my starting location.

**Acceptance criteria:**

Given that I am on the map page,

I can click on the upper text box,

Then I can type in to set my starting location.

**Previous evaluation:**

Date | Outcome | Comments
--- | --- | ---
2020-03-02 | Fail | There should be a drop down of recommendations when typing a location.

**Result:**

To be filled in by product owner upon evaluation.

**Date:**

To be filled in by product owner upon evaluation.

**Signature:**

To be filled in by product owner upon evaluation.

**Comment by mkshiblu** [comment]
I cannot find the upper search bar using the mentioned steps. When I click on the bottom search bar it does bring up the google powered upper search bar which I think overlays the actual in-app top search bar. Is this the expected behavior?

I can see the autosuggestion is working though.


![image](https://user-images.githubusercontent.com/15908043/77976963-66188800-72cc-11ea-9d0a-5310f8f5c14c.png)

![image](https://user-images.githubusercontent.com/15908043/77977035-906a4580-72cc-11ea-9eec-887813cf763b.png)

**Comment by Khal-Shah** [comment]
Yes that is the expected behaviour of how we have currently implemented it as. Once you do pick a location from the auto-complete and then click on the rotated arrow of bottom search bar, the map should take you to the location you chose and display the top search bars, where you can change starting and destination locations again.

**Comment by mkshiblu** [comment]
ACCEPTED. Perhaps a bit confusing for the user because of a total of 3 search bars on different locations, but it seems to be working.
